frontend/src/views/SubjectCourses.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="main-content subject-course">
+    <AppBreadCrumb/>
+    <div class="subject-content">
+      <div class="card-container" v-if="courses.length">
+        <div class="course-card" :key="item.code" v-for="item in courses">
```

Comment:
These course-tiles are nice, but maybe as discussed above should be there own components with a slot where u can dump other things.

Also Wondering whether course cards should support slightly more info? Or is this just placeholder? For courses at the very least we will eventually have meta things like reviews, ratings.

Reply:
I suppose my feeling here is from a UX perspective it looks super similar to the full subject list page and this is mildly confusing.

Reply:
This is kinda placeholder, yes.
Course cards can definitely have some richer stuff, e.g. aggregated ratings, number of reviews etc.
Could be sorted by rating or recommended based on user profile

Reply:
Happy to merge as is. I tend to think we should break out the way it's styled into seperated components (it already is, but you can chunk the common elements).

<TileContainer>
  <Tile>
   <Subject> or <CourseCard>

Reply:
But maybe an errand for another day.

Reply:
Yeah...